### PR TITLE
Bugfixes for atomic host configuration and container deployment

### PR DESCRIPTION
**Pull request by h4xr**

### CODE REVIEW

File: playbooks/satellite/docker-tierdown.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     - name: "Start docker daemon"
       service:
         name: docker
-        state: stopped
+        state: started
```
Comment (jhutar) [inline playbooks/satellite/docker-tierdown.yaml]: Why this change please?

File: playbooks/satellite/roles/docker-container-systemd/files/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,5 @@
-FROM registry.access.redhat.com/rhel7/rhel-tools
-MAINTAINER jeder <jeder@redhat.com>
-RUN yum --disablerepo='*' --enablerepo='rhel-7-server-rpms' install -y openssh-server libselinux-ruby && yum clean all
+FROM registry.access.redhat.com/rhel7/rhel-atomic
+RUN microdnf --enablerepo='rhel-7-server-rpms' install -y openssh-server libselinux-ruby yum subscription-manager
```
Comment (jhutar) [inline playbooks/satellite/roles/docker-container-systemd/files/Dockerfile]: Why we want this? We definitely want yum and so in the container.

File: playbooks/satellite/docker-tierdown.yaml (hunk 1 above)
Comment (h4xr) [reply playbooks/satellite/docker-tierdown.yaml]: @jhutar We need to start the docker container to remove the images.
Before this change, the removal of images was raising the following error "Docker daemon not running"

File: playbooks/satellite/roles/docker-container-systemd/files/Dockerfile (hunk 2 above)
Comment (h4xr) [reply playbooks/satellite/roles/docker-container-systemd/files/Dockerfile]: We don't have yum in rhel-atomic-base image. What this change does is, it installs yum from the rhel-7-server-rpms using microdnf.

So, once our container image is built, we will be having yum command available to us :)

**Comment by h4xr** [comment]
Closing this as abandoned effort
